### PR TITLE
feat(cli): add `fill-analytics` command

### PR DESCRIPTION
**Pull request by Alexandcoats**
This is somewhat dependent on #838

## Notes to Reviewer

As a reviewer, please pay particular attention to the following areas when reviewing this PR and tick the above boxes after you have completed the steps.

#### CLI Changes
* [x] Run chronicle using an INX connection, without analytics enabled.
* [x] Run CLI command `fill-analytics` to fill missing analytics.

### CODE REVIEW

File: src/bin/inx-chronicle/cli.rs [hunk 1]
```diff
@@ -196,4 +235,11 @@ pub enum Subcommands {
     /// Generate a JWT token using the available config.
     #[cfg(feature = "api")]
     GenerateJWT,
+    //#[cfg(feature = "analytics")]
```
Comment (grtlr) [inline src/bin/inx-chronicle/cli.rs:238]: This will probably change once we merge the other PR, right?

**Review by grtlr — APPROVED** [review]

File: src/bin/inx-chronicle/cli.rs (hunk 1 above)
Comment (Alexandcoats) [reply src/bin/inx-chronicle/cli.rs:238]: Yes, that's the idea.

**Review by Alex6323 — APPROVED** [review]

**Comment by Alex6323** [comment]
Looks good, but I cannot test-run the PR atm.

**Review by grtlr — APPROVED** [review]

File: src/bin/inx-chronicle/cli.rs [hunk 2]
```diff
@@ -190,4 +228,10 @@ pub enum Subcommands {
     /// Generate a JWT token using the available config.
     #[cfg(feature = "api")]
     GenerateJWT,
+    #[cfg(feature = "analytics")]
+    FillAnalytics {
+        start_milestone: u32,
```
Comment (grtlr) [inline src/bin/inx-chronicle/cli.rs:233]: Shouldn't we use `MilestoneIndex` here?

**Review by grtlr — APPROVED** [review]

**Pull request merged**